README.mediawiki

```diff
@@ -124,7 +124,7 @@ Those proposing changes should consider that ultimately consent may rest with th
 | URI Scheme
 | Nils Schneider, Matt Corallo
 | Standard
-| Final
+| Replaced
```

Comment:
The background color for this BIP would need to be updated as well (see other entries in the README with Replaced status).